### PR TITLE
Use recommended expression in pragma statement for compiler version

### PR DESCRIPTION
**Pull request by rokso**

### CODE REVIEW

**Comment by rokso** [comment]
@filcap Please review this change, it is very small change in pragma statement

File: contracts/monolithic.sol [hunk 1]
```diff
@@ -22,7 +22,7 @@
     TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
     SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 */
-pragma solidity >=0.4.19;
+pragma solidity ^0.4.19;
```
Comment (filcap) [inline contracts/monolithic.sol]: @rokso , for consistency, can you also make the same changes to the migrations and mock contracts?

Comment (rokso) [reply contracts/monolithic.sol]: @filcap sure

**Comment by rokso** [comment]
@filcap added one more commit as per our discussion, please review.

**Pull request merged**